ios/Podfile.lock

```diff
@@ -282,27 +282,10 @@ PODS:
     - nanopb/encode (= 2.30908.0)
   - nanopb/decode (2.30908.0)
   - nanopb/encode (2.30908.0)
-  - Onfido (29.7.1)
+  - Onfido (29.6.0)
```

Comment:
Lots of unrelated changes in this file 😮 

These are all we need:
![image](https://github.com/Expensify/App/assets/108292595/33f4c52c-900f-4a6a-b7cb-ee94021882ff)
![image](https://github.com/Expensify/App/assets/108292595/4da4cc99-4a53-4f33-8b3b-97727fb37f04)

Please undo all other changes.

Reply:
Those dependencies are coming from the new architecture - it's because of `pod update Onfido`, without that `pod install` isn't going through 🤷

Reply:
Then we might have to test for anything the pod install changed

Reply:
> Those dependencies are coming from the new architecture - it's because of `pod update Onfido`, without that `pod install` isn't going through 🤷

Hmm, it works fine to me without pod update.
If it's required, can we do that in separate PR?